src/otx/core/model/entity/classification.py

```diff
@@ -153,6 +153,11 @@ def _restore_model_forward(self) -> None:
         self.model.forward = func_type(self.original_model_forward, self.model)
         self.original_model_forward = None
 
+    @property
```

Comment:
We have an abstract exporter in `OTXModel`. `ExplainableOTXClsModel` is inherited from `OTXModel` which brings some extra explain functionality. Why should we declare a particular exporter in `ExplainableOTXClsModel`? Following the logic apparently used here, we can implement the exporter right in `OTXModel`.

Reply:
When I first added the model, I naturally assumed that the `OTXModel` would use the `NativeExporter` by default, but it looked something like this 
<img width="618" alt="image" src="https://github.com/openvinotoolkit/training_extensions/assets/38045080/ddbf6e44-4522-4a32-96b5-40f12d2e0313">

This is currently only declared in a common class for MM, and it is expected that custom models will typically be written inheriting from a class like `OTXMulticlassClsModel`, as in the examples in this PR. So I just moved the common things into that `ExplainableOTXClsModel` class. But other tasks might be different, so I didn't move it to `OTXModel` for now.

Reply:
I have no objection to this function being moved out of its proper place.